src/outputs/getEventData.js

```diff
@@ -21,15 +23,24 @@ const outResults = async results => {
         delete x.pos;
         delete x.uuid;
         let time = moment.duration(x.time);
-        x.time = moment({minutes: time.minutes(), seconds: time.seconds()}).format('mm:ss');
+        x.time = moment({ hours: time.hours(), minutes: time.minutes(), seconds: time.seconds()}).format('HH:mm:ss');
+        x.ageGrade = x.ageGrade + '%';
         delete x.firstEvent;
         delete x.pb;
         x.notes = x.notes.join(', ');
 
-        return x;
+        return Object.values(x).join(',');
```

Comment:
`Object.values()` is not deterministic in the ordering of the values, which means you can't guarantee what order the comma separated values are. If you want to enforce an order, you will have to either use a sort function, or hard code the order in an array like `[x.event, x.pos]` etc.